modules/ROOT/pages/getting-started-proxy.adoc

```diff
@@ -1,89 +1,108 @@
-apply policies, configure access to it and more.
+After you create the API asset in Exchange, you import the API asset to API Manager for further configuration.
 
-Follow this steps in order to import your API in API Manager:
+To import your API asset to API Manager:
 
-. In the API Administration Dashboard click in Manage API > Manage API from Exchange
+. Log into Anypoint Platform and select *API Manager*.
 +
-image::getting-started-proxy-manage-api-from-exchange.png[manage-api-from-exchange]
+The API Administration window is displayed.
+. Select *Manage API > Manage API from Exchange*.
+//image::getting-started-proxy-manage-api-from-exchange.png[Manage API from Exchange]
+. On the Manage API from Exchange window, specify the following details:
 +
-. In the next screen, you have to complete the following fields:
+- *API Name*: Type the first characters of the API name that you created in Exchange.
 +
-image::getting-started-proxy-editing-api.png[editing-api,width=75%]
-+
-- *API Name*: Type the first characters of the API name (The same name you chose when you created your asset in Exchange), and
-then API Manager displays APIs to choose from. Select the API one that was recently created in Exchange.
+The list displays the full name of the API asset that you created in the previous procedure.
 +
 - *Asset Type*: For this example we are going to use HTTP, but you can also choose to create a RAML/OAS or WSDL API.
 +
 - *API Version*: Available versions in Exchange.
 +
-- *Managing Type*: You may choose between `Basic Endpoint` or `Endpoint with Proxy`. Here we are using the endpoint with proxy.
+- *Managing Type*: To deploy an API asset with proxy on CloudHUb, select `endpoint with proxy`.
++
+If you do not want to configure your API with proxy, select *Basic endpoint*. 
++
+- *Mule version:* If you are managing an API for Mule 4 or later, select this checkbox. 
 +
-- *Implementation URI*: Enter the URL where the API implementation resides. For this example we are using http://my-implementation.company.com
+In this example, we are creating the API for Mule 4.
 +
-If you are managing an API for Mule 4 or above, please make sure you select the checkbox. In this example, we are creating the API for Mule 4.
+- *Implementation URI*: Enter the URL where the API implementation resides. 
++
+For this example we are using http://my-implementation.company.com:
++
+image::getting-started-proxy-editing-api.png[editing-api,width=75%]
 +
 ----
-If you need further configuration, you may want to click in Advanced Options, and fill in the options available.
+For additional configuration options, click in *Advanced options*, and fill in the details.
 ----
 +
-. Click on `Save`.
+. Click on *Save*.
 
-After that, the API settings view is shown with the current information of the recently imported API.
+The API settings view is displayed with the details of the imported API. Your API instance is now created:
```

Comment:
API, API asset, or API instance? Avoid using more than one term for the same thing.
GLOBAL in all writing.

Reply:
API is a model for example, the API instance is a specific instantiation or application of that API and an API when pushed to or exists in Exchange becomes an asset.